pubspec.yaml

```diff
@@ -1,26 +1,22 @@
 name: w_common
-description: General utilities for Dart projects.
 version: 1.13.0
+description: General utilities for Dart projects.
 author: Workiva Client Platform Team <team-clientplatform@workiva.com>
 homepage: https://www.github.com/Workiva/w_common
 
 environment:
-  sdk: '>=1.12.0 <2.0.0'
+  sdk: '>=1.12.0 <3.0.0'
```

Comment:
Is this lower bound actually correct? I don't think it is because I don't think we got parametric methods until after 1.12.0. Maybe we should just set this to 1.24.3 since Dart 1 is basically deprecated at this point anyway, so there's no good reason to use an old version of it.

Reply:
Makes sense. I'm ok updating it if you are.